ansible/roles/artifacts/tasks/main.yaml

```diff
@@ -269,6 +269,31 @@
     mode: "644"
     checksum: sha256:3d93ca05f31b63424d7d7246a01a2365953705a0ed3323ba5b6fddd744a4bfea
 
+# traffic_light_detector model which referenced by tensorrt_yolox package
```

Comment:
[imho]
We need to clearfy which tlr model will be downloaded in this role.
Do we have our new model name already? How about `one-shot traffic_light_detector model`?

Reply:
@YoshiRi 
> We need to clearfy which tlr model will be downloaded in this role.

all model listed here will be downloaded when setup autoware as default. The used model whill be selected by seting when launching autoware.

> Do we have our new model name already? How about one-shot traffic_light_detector model?

- The current selected name is `yolox_s_car_ped_tl_detector_960_960_batch_1`
- I think all current yolox only support `one-shot` so it seems unnecessary?

Reply:
@YoshiRi do you hav another comments?

Reply:
@badai-nguyen @MasatoSaeki Now I understood. LGTM